FRC1710-2019/src/main/java/frc/robot/Constants.java

```diff
@@ -28,5 +29,16 @@
     public static int ticksPerInch = 217;
 
     //drive to position slow down rate
-    public static double slowDownPercent = 0.1;
+	public static double slowDownPercent = 0.1;
+	
+	//vision constants
+	public static double kpAim = 0.005;
+	public static double Hdiff = 7.79;
+	public static double MAX = 10.0;
+	public static double con = .06;
+
+	public static double heightOfLimeLight = 19.0;
+	public static double heightOfTarget = 29.0;
```

Comment:
Wait if we are defining Hdiff, then what are these used for? Can they be removed? (Although I like how good the variable names are here)